src/main/scala/org/hammerlab/guacamole/commands/GermlineAssemblyCaller.scala

```diff
@@ -104,12 +104,18 @@ object GermlineAssemblyCaller {
       val referenceKmerSink = currentReference.takeRight(kmerSize)
 
       // If the current window size doesn't cover the kmer size we won't be able to find the reference start/end
-      if (currentReference.size < kmerSize)
+      if (currentReference.size < kmerSize || referenceKmerSource == referenceKmerSink)
```

Comment:
Yes, definitely possible, but as this is setup, I don't see a good way to handle it for now. Possible solutions may be just increasing the kmer size and looking again, but just punting on it for now.